### PR TITLE
Browse/search result design

### PR DESCRIPTION
**Pull request by yzhang54**

### CODE REVIEW

File: src/app/pages/browse/browse-archive/browse-archive.component.ts [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 })
 export class BrowseArchiveComponent implements OnInit {
-
-  constructor() { }
+  constructor(private route: ActivatedRoute) {}
 
   ngOnInit(): void {
+    this.route.queryParams.subscribe((params) => {
+    //this.name = params['name'];
+    });
```
Comment (JOOOEG123) [inline src/app/pages/browse/browse-archive/browse-archive.component.ts:15]: you are close to the approach I am going with.

File: src/app/pages/browse/main-browse/main-browse.component.ts [hunk 2]
```diff
@@ -44,6 +47,15 @@ export class MainBrowseComponent implements OnInit, OnDestroy {
     this.lettersBtnClick('All');
   }
 
+  onViewChange() {
+    console.log("onViewChange clicked")
+    var select_option = (<HTMLInputElement>document.getElementById("viewSelect")).value;
+    var search_layout = document.getElementsByName("searchLayout");
```
Comment (JOOOEG123) [inline src/app/pages/browse/main-browse/main-browse.component.ts:53]: You don't get value like this in a modern framework.

Comment (JOOOEG123) [reply src/app/pages/browse/main-browse/main-browse.component.ts:53]: you can just use variable **curView**.

**Review by JOOOEG123 — CHANGES_REQUESTED** [review]

**Review by weilonp — APPROVED** [review]

File: src/app/pages/browse/memoir/memoir.component.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  xmlns="http://www.w3.org/2000/svg"
+  width="16"
+  height="16"
+  fill="currentColor"
+  class="bi bi-share"
+  viewBox="0 0 16 16"
+>
+  <path
+    d="M13.5 1a1.5 1.5 0 1 0 0 3 1.5 1.5 0 0 0 0-3zM11 2.5a2.5 2.5 0 1 1 .603 1.628l-6.718 3.12a2.499 2.499 0 0 1 0 1.504l6.718 3.12a2.5 2.5 0 1 1-.488.876l-6.718-3.12a2.5 2.5 0 1 1 0-3.256l6.718-3.12A2.5 2.5 0 0 1 11 2.5zm-8.5 4a1.5 1.5 0 1 0 0 3 1.5 1.5 0 0 0 0-3zm11 5.5a1.5 1.5 0 1 0 0 3 1.5 1.5 0 0 0 0-3z"
+  />
```
Comment (JOOOEG123) [inline src/app/pages/browse/memoir/memoir.component.html:27]: move create a svg file for this and use img. Reson: makes it easy to maintain all images

File: src/app/pages/browse/share/share.component.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="M4.715 6.542 3.343 7.914a3 3 0 1 0 4.243 4.243l1.828-1.829A3 3 0 0 0 8.586 5.5L8 6.086a1.002 1.002 0 0 0-.154.199 2 2 0 0 1 .861 3.337L6.88 11.45a2 2 0 1 1-2.83-2.83l.793-.792a4.018 4.018 0 0 1-.128-1.287z"
+                />
+                <path
+                  d="M6.586 4.672A3 3 0 0 0 7.414 9.5l.775-.776a2 2 0 0 1-.896-3.346L9.12 3.55a2 2 0 1 1 2.83 2.83l-.793.792c.112.42.155.855.128 1.287l1.372-1.372a3 3 0 1 0-4.243-4.243L6.586 4.672z"
+                />
+              </svg>
+            </span>
```
Comment (JOOOEG123) [inline src/app/pages/browse/share/share.component.html:50]: move it to svg

**Review by JOOOEG123 — CHANGES_REQUESTED** [review]
Please fix the issue in your comment before you work on a different page or functionality. Reason: It is much easy and less confusing if you focus one one thing at a time.

File: src/app/pages/browse/memoir/memoir.component.html (hunk 3 above)
Comment (yzhang54) [reply src/app/pages/browse/memoir/memoir.component.html:27]: @weilonp Hi, could you please work on this as you did this part?

**Pull request merged**